src/Nethermind/Nethermind.BeaconNode.OApiClient/BeaconNodeProxy.cs

```diff
@@ -107,6 +107,43 @@ public async Task<ApiResponse<Syncing>> GetSyncingAsync(CancellationToken cancel
             return ApiResponse.Create((StatusCode) (int) httpResponse.StatusCode, content);
         }
 
+        public async Task<ApiResponse<Attestation>> NewAttestationAsync(BlsPublicKey validatorPublicKey,
+            bool proofOfCustodyBit, Slot slot, CommitteeIndex index,
+            CancellationToken cancellationToken)
+        {
+            string baseUri = "validator/attestation";
+
+            // NOTE: Spec 0.10.1 still has old Shard references in OAPI, although the spec has changed to Index;
+            // use Index as it is easier to understand (i.e. the spec OAPI in 0.10.1 is wrong)
+
+            Dictionary<string, string> queryParameters = new Dictionary<string, string>
+            {
+                ["validator_pubkey"] = validatorPublicKey.ToString(),
+                ["poc_bit"] = proofOfCustodyBit ? "1" : "0",
+                ["slot"] = slot.ToString(),
+                ["index"] = index.ToString()
+            };
+
+            string uri = QueryHelpers.AddQueryString(baseUri, queryParameters);
+
+            using HttpResponseMessage httpResponse =
+                await _httpClient.GetAsync(uri, HttpCompletionOption.ResponseHeadersRead, cancellationToken);
+
+            int statusCode = (int) httpResponse.StatusCode;
+            if (statusCode == (int) StatusCode.InvalidRequest
+                || statusCode == (int) StatusCode.CurrentlySyncing)
```

Comment:
why are you casting it to int? 

statusCode = httpResponse.StatusCode;
statusCode == StatusCode.InvalidRequest -> isn't that the same?
and after that you are casting it to StatusCode again in return